### ISSUE TITLE
Proposition de @juliedelauge

### ISSUE THREAD

**Issue by lvahe** [cbn-alpin/silene-geonature#96]
Proposition de @juliedelauge 
![Capture d’écran 2021-01-15 à 12 18 12](https://user-images.githubusercontent.com/64422654/104718811-c8435c00-572b-11eb-966c-7eed29cd2b5f.png)

Colonne jeux de données : 

- raccourcir les textes en supprimant "jeu de données de ..."
- ne mettre que le nom de l'organisme

@jpm-cbna : est-ce lié aux métadonnées des JDD et donc probablement modifié lorsque les JDD finaux seront intégrés à la Synthèse ?

_Originally posted by @lvahe in https://github.com/cbn-alpin/silene-geonature/issues/10#issuecomment-760885000_

**Comment by jpm-cbna** [comment]
Oui, le nom du jeu de données tel que définit dans les métadonnées s'affiche dans cette colonne.

**Comment by juliedelauge** [comment]
Il s’agit dans le data set de
name [VARCHAR(255)]

ou
#    shortname

Julie DELAUGE
Adjointe à la Direction - Responsable Connaissance et Programmes
[cid:image001.jpg@01D6EB51.D27A5840]
CEN PACA
Nouvelle adresse : 18 avenur du Gand – 04200 Sisteron
Tél. 04 92 34 40 10
julie.delauge@cen-paca.org<mailto:julie.delauge@cen-paca.org>
www.cen-paca.org<http://www.cen-paca.org/>


De : Jean-Pascal Milcent <notifications@github.com>
Envoyé : vendredi 15 janvier 2021 14:25
À : cbn-alpin/silene-geonature <silene-geonature@noreply.github.com>
Cc : Julie Delauge <julie.delauge@cen-paca.org>; Mention <mention@noreply.github.com>
Objet : Re: [cbn-alpin/silene-geonature] [GeoNature]Tableau résultats de recherche : configuration (#96)


Oui, le nom du jeu de données tel que définit dans les métadonnées s'affiche dans cette colonne.

**Comment by jpm-cbna** [comment]
C'est bien le champ `name` du [fichier d'import DATASET ](https://wiki-sinp.cbn-alpin.fr/database/import-formats#description_du_format_dataset) dans le format d'échange.

Ce qui correspond au champ `gn_meta.t_dataset.dataset_name` dans la base GeoNature.
